bears/c_languages/GNUIndentBear.py

```diff
@@ -155,6 +156,10 @@ def create_arguments(filename, file, config_file,
             Any command line options the indent binary understands. They
             will be simply passed through.
         """
+        # The limit is set to a number which equals to the max size in Python
```

Comment:
It isnt the max size in Python.

See https://docs.python.org/3/library/sys.html#sys.maxsize

Note that `Py_ssize_t` isnt something that is exposed to the Python 3 language at all -- it is quite internal, so it is better to say this is an arbitrary high number.

Reply:
Okay. Changing it